### ISSUE TITLE
Figure out roadmap for presentation

### ISSUE THREAD

**Issue by somexpert** [ldko/gitlab-collab#1]
We should start figuring individual pieces we want to get done. We can maybe create a board to show what has yet to be started, what is in-progress, and what has been completed. What do you think @ldko?

**Comment by somexpert** [comment]
I guess the term here is "Project" rather than "board"

**Comment by ldko** [comment]
It's a "project board" :). Yes, please create one, thank you!

**Comment by ldko** [comment]
Things we may want to include (this is from the notes I jotted down at our meeting and should not be considered must-do):

- Show how to create a repo and add a file
- Talk about using issues, project board, labels, milestone?, assigning to a person
- Opening an MR or PR for reviewing work
- Using the provided wiki
- gists/snippets, attach a file
- permissions and visibility of project
- groups/organization, adding members, restricting access to them
- Demo:
-- create a project
-- Set up issues with labels and assignees
-- Create Readme, save, do more changes and do a preview
-- Open a MR/PR
-- Look at Contributors, graph
-- Look at board
-- Set milestones
- Mention attending git section of SWC workshop?

**Comment by somexpert** [comment]
I think those are a solid starting point, and we can cut/add as necessary. Maybe we can create individual issues for each topic? That would allow discussion for each as well as giving us a good way to start adding slides without worrying about order and flow of the overall presentation yet. Could have an MR/PR per issue, which would be the slides added to cover that individual topic. What do you think?

**Comment by ldko** [comment]
I think that makes sense. We could open the specific issue and assign ourselves when we start working on one part and have a strong sense of what that slide(s) will be (rather than create a bunch of issues now that may end up changing a lot).

**Comment by somexpert** [comment]
So I think we can get the wiki stuff out of the way by having it play the role of "style guide" for our slides, or maybe being an overview to layout the order of topics for our presentation (or both). If you like that I can create an issue for it for specific discussions and we can move it through the project board.

**Comment by ldko** [comment]
yeah a wiki with a page for style guide and order of topics/table of contents sounds like a good start. Creating an issue sounds fine.

**Comment by somexpert** [comment]
I guess if we want a milestone we should also get that started, so we can add related issues to it.
